### ISSUE TITLE
java.lang.SecurityException

### ISSUE THREAD

**Issue by tom-krusch** [getsentry/sentry-react-native#903]
**OS:**
- [ ]  Windows
- [x]  MacOS
- [ ]  Linux

**Platform:**
- [ ]  iOS
- [x]  Android

**SDK:**
- [x]  `@sentry/react-native` (>= 1.0.0)
- [ ]  `react-native-sentry` (<= 0.43.2)

**SDK version:** 1.3.9

**`react-native` version:** 0.62.0

**Are you using Expo?**
- [ ]  Yes
- [x]  No

**Are you using sentry.io or on-premise?**
- [ ]  sentry.io (SaaS)
- [x]  on-premise

**Configuration:**

(`@sentry/react-native`)
```
Sentry.init({
  dsn: 'https://...@...'});
```

---
**I have following issue:**

on Android Version 5 and 5.1 there is an error on initialization of sentry on the live build

**Actual result:**

```
java.lang.SecurityException: Neither user 10107 nor current process has android.permission.READ_PHONE_STATE.

java.lang.SecurityException: 
  at android.os.Parcel.readException (Parcel.java:1549)
  at android.os.Parcel.readException (Parcel.java:1502)
  at com.android.internal.telephony.ITelephonyRegistry$Stub$Proxy.listenForSubscriber (ITelephonyRegistry.java:544)
  at android.telephony.TelephonyManager.listen (TelephonyManager.java:2658)
  at io.sentry.android.core.PhoneStateBreadcrumbsIntegration.register (PhoneStateBreadcrumbsIntegration.java:52)
  at io.sentry.core.Hub.<init> (Hub.java:38)
  at io.sentry.core.Sentry.init (Sentry.java:144)
  at io.sentry.core.Sentry.init (Sentry.java:92)
  at io.sentry.android.core.SentryAndroid.init (SentryAndroid.java:59)
  at io.sentry.android.core.SentryAndroid.init (SentryAndroid.java:44)
  at io.sentry.RNSentryModule.startWithDsnString (RNSentryModule.java:66)
  at java.lang.reflect.Method.invoke (Method.java)
  at java.lang.reflect.Method.invoke (Method.java:372)
  at com.facebook.react.bridge.JavaMethodWrapper.invoke (JavaMethodWrapper.java:371)
  at com.facebook.react.bridge.JavaModuleWrapper.invoke (JavaModuleWrapper.java:150)
  at com.facebook.react.bridge.queue.NativeRunnable.run (NativeRunnable.java)
  at android.os.Handler.handleCallback (Handler.java:739)
  at android.os.Handler.dispatchMessage (Handler.java:95)
  at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage (MessageQueueThreadHandler.java:26)
  at android.os.Looper.loop (Looper.java:135)
  at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run (MessageQueueThreadImpl.java:225)
  at java.lang.Thread.run (Thread.java:831)
```

**Comment by HazAT** [comment]
@marandaneto Did something change in the permissions?

**Comment by marandaneto** [comment]
Hey @tom-krusch thanks for raising this.

this issue is already fixed by getsentry/sentry-android/pull/415 and it crashes only on older android versions (if missing permission).

upgrade the SDK to at least https://github.com/getsentry/sentry-react-native/releases/tag/1.4.0
or the latest, even better https://github.com/getsentry/sentry-react-native/releases/tag/1.4.2

or, a workaround would be to add an extra permission to your manifest, which is only required, as I said, by older versions.

`<uses-permission android:name="android.permission.READ_PHONE_STATE" />`

This should solve the issue, sorry about that, and thanks for reporting once more.
